packages/core/src/external/opensearch/file/file-searcher-direct.ts

```diff
@@ -19,12 +20,14 @@ export class OpenSearchFileSearcherDirect implements OpenSearchFileSearcher {
 
     debug(`Searching on index ${indexName}...`);
     const queryPayload = {
+      size: 1_000,
       query: {
         bool: {
           must: [
             {
-              query_string: {
-                query,
+              // https://docs.opensearch.org/docs/latest/query-dsl/full-text/simple-query-string/
+              simple_query_string: {
+                query: cleanupQuery(query),
```

Comment:
This is less restrictive, so we get fewer errors in case of a bad input from the user.